hooks/blueprint

```diff
@@ -5,7 +5,7 @@ declare -a merge
 
 merge=( \
   cf-deployment/cf-deployment.yml \
-  cf-deployment/operations/rename-network-and-deployment.yml
+  # cf-deployment/operations/rename-network-and-deployment.yml
```

Comment:
This should be an option, as we have users that just want to use the default (or another single network)